src/database/models/index.js

```diff
@@ -1,30 +1,39 @@
-import pg from 'pg';
-import dotenv from 'dotenv';
+import { readdirSync } from 'fs';
+import { basename as _basename, join } from 'path';
+import { Sequelize } from 'sequelize';
 
-dotenv.config();
-const DATABASE_URI = `postgres://${process.env.DEV_DB_USERNAME}:${process.env.DEV_DB_PASSWORD}@${process.env.DEV_DB_HOSTNAME}:${process.env.DEV_DB_PORT}/${process.env.DEV_DB_NAME}`;
-const pool = new pg.Pool({
-  connectionString: DATABASE_URI,
-});
+require('dotenv').config();
 
-pool.on('connect', () => {
-});
+const basename = _basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require('../config.js')[env];
```

Comment:
Please use ES6

Reply:
@muchristian at what line are you referring to exactly?